requirements.txt

```diff
@@ -12,5 +12,6 @@ imageio>=2.5.0
 imageio-ffmpeg
 colorcet
 cmocean
-scooby>=0.5.0
+scooby>=0.5.1
```

Comment:
Also in `setup.py`

And we will have to remember to bump this on the conda forge recipe for the next release

Reply:
I'm clearly out of practice with PRs to `pyvista`...

Reply:
haha, there's too much to keep track of